packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -442,7 +479,7 @@ describe('KeyringController', () => {
                 password,
                 currentSeedWord,
               );
-              expect(initialState).toStrictEqual(controller.state);
+              expect(initialState.vault).toStrictEqual(controller.state.vault);
```

Comment:
The encryptor is mocked so the vault will not be the result of a real encryption, nor related to the actual content in any way

Reply:
We could verify what is being passed to `encryptor.encrypt` instead